exist-core/src/main/java/org/exist/xquery/functions/util/PrologFunctions.java

```diff
@@ -47,7 +47,8 @@
 				new FunctionParameterSequenceType("prefix", Type.STRING, Cardinality.EXACTLY_ONE, "The prefix to be assigned to the namespace"),
 				new FunctionParameterSequenceType("location", Type.ANY_URI, Cardinality.EXACTLY_ONE, "The location of the module")
 			},
-			new SequenceType(Type.ITEM, Cardinality.EMPTY)),
+			new SequenceType(Type.ITEM, Cardinality.EMPTY),
+			"Prefer fn:load-xquery-module"),
```

Comment:
It is better to link by class reference that name, e.g. change `"Prefer fn:load-xquery-module"` ->`LoadXQueryModule.class`